<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBIR</class>
 <widget class="QMainWindow" name="CBIR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CBIR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbSetLibDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择库目录</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSetLibFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择库文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLibDir">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLibFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBLibDir">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBLibFile">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBBuildLib">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBLoadLib">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbQueryImage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbSetQuery">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择查询图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBSetQuery">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBSearch">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBReSearch">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ReSearch</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbRes1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbRes2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbRes3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbRes4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBUp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBNext">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>23</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
